src/t_thmap.c

```diff
@@ -242,13 +242,13 @@ alloc_test_wrapper(size_t len)
 	uintptr_t p = space_allocated;
 	space_allocated += roundup2(len, sizeof(void *));
 	assert(space_allocated <= sizeof(space));
-	return p;
+	return p + 4;
```

Comment:
Can you please define something like `static const unsigned	space_off = 4;` after `space_allocated` and use the constant? Thanks.